xmlrpc/views.py

```diff
@@ -56,6 +56,6 @@ def xmlrpc_handler(request):
     xmlrpc_handler.register_instance(PublicFunctions())
   
     response = HttpResponse(mimetype='application/xml')
-    response.write(xmlrpc_handler._marshaled_dispatch(request.raw_post_data))
+    response.write(xmlrpc_handler._marshaled_dispatch(request.body))
```

Comment:
I could reproduce the error before this change. And can verify that it works after applying the fix. Also, since this is just a name refactor, documented in the Django docs, I would say this is good to go.